compiler/x/amd64/codegen/AMD64SystemLinkage.cpp

```diff
@@ -724,6 +724,7 @@ TR::Register *
 TR_AMD64SystemLinkage::buildIndirectDispatch(TR::Node *callNode)
    {
    TR::SymbolReference *methodSymRef = callNode->getSymbolReference();
+   TR_ASSERT(methodSymRef->getSymbol()->castToMethodSymbol()->isComputedStatic(), "system linkage only supports computed static call for now %p\n", callNode);
```

Comment:
Yea, I agree. Added the same assert for ppc, x86-32 bits doesn't support indirect call at all and there is an assertion for that already. I also changed the assert to isComputed() because for codegen computedStatic and computedVirtual are the same. It is regular virtual call vs computed call that matters.